scripts/plot_fit.py

```diff
@@ -67,6 +65,30 @@ def main(args):
     plt.savefig(folder+'/curve.png')
 
 
+def main(args):
+    folder = args.file
+
+    splits = ['train', 'validation', 'test']
+
+    ### plot predictions fit
```

Comment:
We only need one `#` ideally with a capital afterwards and a period at the end of the comment.